mne/bem.py

```diff
@@ -332,3 +336,109 @@ def cost_fun(x, points):
     origin = x_opt[:3]
     radius = x_opt[3]
     return radius, origin
+
+
+# ############################################################################
+# Create BEM surfaces
+
+@verbose
+def make_watershed_bem(subject=None, subjects_dir=None, overwrite=False,
+                       volume='T1', atlas=False, gcaatlas=False, preflood=None,
+                       verbose=None):
+    """
+    Create BEM surfaces using the watershed algorithm included with FreeSurfer
+
+    Parameters
+    ----------
+    subject : string
+        Subject name (required)
+    subjects_dir : string
+        Directory containing subjects data. If None use
+        the Freesurfer SUBJECTS_DIR environment variable.
+    overwrite : bool
+        Write over existing files
+    volume : string
+        Defaults to T1
+    atlas : bool
+        Specify the --atlas option for mri_watershed
+    gcaatlas : bool
+        Use the subcortical atlas
+    preflood : int
+        Change the preflood height
+    verbose : bool, str or None
+        If not None, override default verbose level
```

Comment:
still need this bit (I assume you're waiting to make all changes in one more pass, but just wanted to remind you)